metadata.yaml

```diff
@@ -1,23 +1,50 @@
-# Copyright 2021 jose
+# Copyright 2021 Canonical Ltd.
 # See LICENSE file for licensing details.
 
 # For a complete list of supported options, see:
 # https://discourse.charmhub.io/t/charm-metadata-v2/3674/15
-name: loki-operator
-display-name: |
-  TEMPLATE-TODO: fill out a display name for the Charmcraft store
+
+name: loki-k8s
+display-name: Loki
 description: |
-  TEMPLATE-TODO: fill out the charm's description
+  Loki for Kubernetes cluster
 summary: |
-  TEMPLATE-TODO: fill out the charm's summary
+  Unlike other logging systems, Loki is built around the idea of only indexing metadata
+  about your logs: labels (just like Prometheus labels).
+  Log data itself is then compressed and stored in chunks in object stores such as S3 or GCS,
+  or even locally on the filesystem. A small index and highly compressed chunks simplifies
+  the operation and significantly lowers the cost of Loki.
+
+maintainers:
+    - Jose Massón <jose.masson@canonical.com>
 
-# TEMPLATE-TODO: replace with containers for your workload (delete for non-k8s)
 containers:
-  httpbin:
-    resource: httpbin-image
+  loki:
+    resource: loki-image
+    mounts:
+      - storage: active-index-directory
+        location: /loki/boltdb-shipper-active
+      - storage: loki-chunks
+        location: /loki/chunks
+
+storage:
+  active-index-directory:
+    type: filesystem
+    description: Mount point in which Loki will store index
+  loki-chunks:
+    type: filesystem
+    description: Mount point in which Loki will store chunks (objects)
+
+provides:
+  logging:
+    interface: loki-push-api
+
+requires: # change
```

Comment:
@Abuelodelanada we also need to be able to send alerts to Alertmanager. The `loki_push_api` relation interface should allow the charms sending us logs to define `alert_rules`, the same way it works in `prometheus_scrape`.